pkg/xds/generator/egress/external_services_generator.go

```diff
@@ -0,0 +1,195 @@
+package egress
+
+import (
+	"sort"
+
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
+	core_mesh "github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
+	core_xds "github.com/kumahq/kuma/pkg/core/xds"
+	xds_context "github.com/kumahq/kuma/pkg/xds/context"
+	envoy_common "github.com/kumahq/kuma/pkg/xds/envoy"
+	envoy_clusters "github.com/kumahq/kuma/pkg/xds/envoy/clusters"
+	envoy_listeners "github.com/kumahq/kuma/pkg/xds/envoy/listeners"
+	"github.com/kumahq/kuma/pkg/xds/envoy/tls"
+)
+
+type ExternalServicesGenerator struct {
+}
+
+// Generate will generate envoy resources for one, provided in ResourceInfo mesh
+func (g *ExternalServicesGenerator) Generate(
+	_ xds_context.Context,
```

Comment:
can we remove xds context then from generators? It is not used in any of them

Reply:
addressed